### ISSUE TITLE
UnnecessaryStringOutput false positive when text starts with # character

### ISSUE THREAD

**Issue by kluka-CS** [sds/haml-lint#344]
I am encountering another edge case for the mentioned rule / linter

```haml
%h4= "#{some_value}: #{number_to_currency(some_other_value}"
```

haml-lint then reports
```
UnnecessaryStringOutput: `= "..."` should be rewritten as `...`
```

However when I remove the quotation marks, haml evaluates as a comment (I guess). So then I have no content anymore

**Comment by kwtuku** [comment]
I think you need to remove the equals sign `=` as well as the quotation marks `""`.

```haml
%h4 #{some_value}: #{number_to_currency(some_other_value)}
```

**Comment by sds** [comment]
Realizing I misread and mislabeled this a long time ago.

Confirming what has already been said: you need to remove the `=` and double quotes. The lint is there to tell you that they aren't necessary. 

Closing as intended behavior.
